### PR TITLE
docs: fix broken link in navigation readme

### PR DESCRIPTION
**Pull request by fnKode**

### CODE REVIEW

File: .changeset/cool-spies-return.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@equinor/mad-navigation": patch
+---
+
+Fix broken link
```
Comment (tormodAase) [inline .changeset/cool-spies-return.md:5]: Probably don't need a changeset for this one

Comment (tormodAase) [reply .changeset/cool-spies-return.md:5]: Or maybe the npm registry wont update the description unless we version bump? @HaakonSvane

**Review by tormodAase — COMMENTED** [review]
Not sure if we need a changeset. Maybe @HaakonSvane can give some input on whether we need a changeset or not

File: .changeset/cool-spies-return.md (hunk 1 above)
Comment (HaakonSvane) [reply .changeset/cool-spies-return.md:5]: Yeah it probably won't see this package as publishable unless we do I think

**Review by HaakonSvane — APPROVED** [review]

**Pull request merged**